### PR TITLE
Close the file after reading in verifydependencies#main

### PR DESCRIPTION
**Pull request by tedyu**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
For verifydependencies, in the main() func, after the file is done being read, we should close it.

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by tedyu** [comment]
cc @neolit123

**Review by justaugustus — CHANGES_REQUESTED** [review]
@tedyu -- hey, can you update the commit message and PR title to be explicit about what file you're referring to?

**Comment by tedyu** [comment]
@justaugustus 
I have updated commit message, the subject and the description.

**Comment by justaugustus** [comment]
Thanks!
/lgtm
/approve

File: cmd/verifydependencies/verifydependencies.go [hunk 1]
```diff
@@ -72,23 +72,26 @@ func main() {
...
-			}
-			matcher := regexp.MustCompile(refPath.Match)
-			depFileScanner := bufio.NewScanner(file)
-			var found bool
-			for depFileScanner.Scan() {
-				line := depFileScanner.Text()
-				if matcher.MatchString(line) && strings.Contains(line, dep.Version) {
-					found = true
-					break
+			func() {
```
Comment (neolit123) [inline cmd/verifydependencies/verifydependencies.go]: nit: i would have declared a function for this instead of using a lambda.
/lgtm

**Pull request merged**